### ISSUE TITLE
Support tvOS?

### ISSUE THREAD

**Issue by gwangpa** [algolia/instantsearch-ios#102]
**Is your feature request related to a problem? Please describe 🙏**
Not really, but I have to implement http request manually which needs lot of effort.

**Describe the solution you'd like 🤔**
Support tvOS, so I can make request easily as iOS does.

**Describe alternatives you've considered ✨**
Manually create http request, but not an ideal.

**Additional context**
I'm using cocoapods to install this SDK

Thank you!

**Comment by spinach** [comment]
Hi @gwangpa,

Thank you for contacting us! 

So we have 3 main libraries in place: The InstantSearchClient, InstantSearchCore, and InstantSearchUI. the first 2 do support any Apple platform (including tvOS) and contain all the networking request and business logic. So you can basically use them.

Here is an example of how you would go about it for a `refinement list`:
In the Refinement list [docs](https://www.algolia.com/doc/api-reference/widgets/refinement-list/ios/), you can see the following: 

- Searcher: Part of InstantSearch Core (available for you to use)
- FilterState: Part of InstantSearch Core (available for you to use)
- FacetListInteractor: Part of InstantSearch Core (available for you to use)
- **FacetListController: This is what you'll need to implement with tvOS views** 
- FacetListPresenter: Optional. Part of InstantSearch Core (available for you to use)

Let us know if that makes sense!

**Comment by VladislavFitz** [comment]
Hi @gwangpa,

We are about to release the new version of InstantSearch providing tvOS support.
To install it via CocoaPods, please add the following line to your `Podfile`:
```ruby
pod 'InstantSearch', '~> 7.0.0-beta.4'
```
 Let us know if you have any trouble or question about integration of InstantSearch into your tvOS project. We would also appreciate to hear your feedback on it.

**Comment by gwangpa** [comment]
Hi @VladislavFitz, @spinach thanks for replying.

My project is  already integrated with `InstantSearch` version `4.0` and I'm trying to integrate with tvOS. 

I've tested to beta version as @VladislavFitz said. I can install pods for iOS and tvOS target, but it's no longer able to find `Client`. Maybe this is because of version differences or different library.

I found another solution as @spinach said. I just use `InstantSearchCore` version `6.6.3` and it seems working well. I will stick with this approach.

Thanks!
